services/api/src/resources/task/helpers.ts

```diff
@@ -99,7 +99,7 @@ export const Helpers = (sqlClientPool: Pool, hasPermission, adminScopes) => {
       let rows = await query(sqlClientPool, Sql.selectTask(insertId));
       const taskData = R.prop(0, rows);
 
-      pubSub.publish(EVENTS.TASK, taskData);
+      pubSub.publish(EVENTS.TASK, {taskChanged: taskData});
```

Comment:
The data payload here is different from all other `pubSub.publish` calls and it appears that adding tasks doesn't trigger a subscription.

```suggestion
      pubSub.publish(EVENTS.TASK, taskData);
```